Sources/SwiftLanguageService/SignatureHelp.swift

```diff
@@ -36,16 +36,23 @@ fileprivate extension String {
 }
 
 fileprivate extension ParameterInformation {
-  init?(_ parameter: SKDResponseDictionary, _ signatureLabel: String, _ keys: sourcekitd_api_keys) {
+  init?(
```

Comment:
Could you add a short comment what the key and value of `parameterDocumentation` is?